Sources/Client/Client_Update.cpp

```diff
@@ -936,6 +936,12 @@ namespace spades {
 					Handle<IAudioChunk> c =
 					  audioDevice->RegisterSound("Sounds/Weapons/Spade/HitPlayer.opus");
 					audioDevice->Play(c, hitPos, AudioParam());
+				} else if(type == HitTypeHead) {
+					Handle<IAudioChunk> c =
+					  audioDevice->RegisterSound("Sounds/Weapons/Impacts/whack.wav");
+					AudioParam param;
+					param.volume = 30.f;
+					audioDevice->Play(c, hitPos, param);
```

Comment:
I suggest using `PlayLocal(c, param)` instead of `Play(c, hitPos, param)` since it is a sound feedback provided as a reward for a successful headshot, and is not a sound caused by others' actions. In this case, you might want to lower the volume to less than `1` since it won't be no longer attenuated by the distance.

Reply:
I don't think playing it locally is a good idea, exactly because of the fact that it's not attenuated by distance, since it is attenuated by distance in AoS

Reply:
I also don't remember hearing the sound when a player is getting shot not by me
I didn't use PlayLocal because as @NotAFile said volume doesn't change with distance and it sounds weird